scripts/dev_env/2_1_ubuntu_18_04.sh

```diff
@@ -0,0 +1,60 @@
+NATIF_AVAILABLE="true"
+DOCKER_AVAILABLE="true"
+
+ALL_NATIF_PACKAGES='build-essential
+    clang-format-6.0
+    cmake
+    curl
+    libboost-all-dev
+    libgeos-dev
+    libgoogle-perftools-dev
+    liblog4cplus-dev
+    libosmpbf-dev
+    libpqxx-dev
+    libproj-dev
+    libprotobuf-dev
+    libssl-dev
+    libzmq3-dev
+    postgis
+    postgresql
+    protobuf-compiler
+    python
+    python-pip
+    python-virtualenv
+    rabbitmq-server
+    virtualenvwrapper'
+
+ALL_DOCKER_PACKAGES='docker.io
+    postgis
+    postgresql
+    python
```

Comment:
Python is already in `ALL_NATIF_PACKAGES` ?

Reply:
It's either ALL_DOCKER_PACKAGES or ALL_NATIF_PACKAGES